src/telegram_py_bot.py

```diff
@@ -11,10 +27,12 @@ async def qr(update, context):
     QR(update.message.text)
```

Comment:
Prova a rinominare la funzione in riga 26 da `qr` a `qr_generation_function` ad esempio, credo che vada in conflitto con l'oggetto `qr` dentro la funzione `QR` definita sopra.

Reply:
fatto